tests/update/updates_packagekit_gpk.pm

```diff
@@ -52,10 +52,15 @@ sub run {
             assert_screen \@updates_tags, 100;
         }
 
+        if (match_has_tag("updates_restart_application")) {
+            assert_and_click("updates_restart_application");
+        }
+
         if (match_has_tag("updates_none")) {
-            send_key "ret";
+            send_key 'ret';
             return;
         }
+
```

Comment:
Please don't add a random blank line. Different people have different preferences and the next guy might just want to remove it again. Also especially here you are hiding that the 'if/else' continues with 'elsif'.

Reply:
If I remove this it will be inconsistent with lines 58 and 54

Reply:
No inconsistency, because we have an **elsif** on next line.